cmd/compose/alpha.go

```diff
@@ -25,6 +25,10 @@ func alphaCommand(p *ProjectOptions, backend api.Service) *cobra.Command {
 		Short:  "Experimental commands",
 		Use:    "alpha [COMMAND]",
 		Hidden: true,
+		Annotations: map[string]string{
+			"experimental":    "true",
```

Comment:
I think we only need the `experimentalCLI` part, as this one may (not sure if that's implemented in compose) hide the command based on whether the daemon has experimental features enabled.